homeassistant/components/wallbox/strings.json

```diff
@@ -25,5 +25,63 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
+  },
+  "entity": {
+    "lock": {
+      "lock": {
+        "name": "[%key:component::lock::title%]"
```

Comment:
@hesselonline Could you maybe give your opinion on the name change for the lock and the switch? They used to be locked/unlocked and pause/resume, but for me it didn't really describe what did it lock or what did it pause or resume. In this PR it would change to `{wallbox name} lock` and `{wallbox name} pause charging`. But I don't own a Wallbox so I would value your input on this one :)

Reply:
I believe this is a good alternative for the lock title; fine by me.